mantidproject/mantid
# Indirect FQ simultaneous fitting

**Description of work.**

This adds the simultaneous fitting to the FQ tab in mantid. 



**To test:**
  * If testing this PR slack me and I'll send you the relevant data file.
  * Load data file into FQ tab and try the a couple of relevant fits. Check that fits look correct.
  * Try switching between full function tab and template function tab.
  * Try some weird things to try and break the GUI



Fixes #26881  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
